backend/internal/storage/postgres/schema/user.go

```diff
@@ -328,6 +328,103 @@ func (r *UserRepository) GetUserFeed(ctx context.Context, id uuid.UUID) ([]*mode
 
 }
 
+func (r *UserRepository) CreateSection(ctx context.Context, sectiontype models.SectionType) (models.SectionType, error) {
+
+	if err := r.db.QueryRow(ctx, `INSERT INTO "section_type" (id, title, search_type) VALUES ($1, $2, $3) RETURNING id`, sectiontype.ID, sectiontype.Title, sectiontype.SearchType).Scan(&sectiontype.ID); err != nil {
+		return models.SectionType{}, err
+	}
+	return sectiontype, nil
+}
+
+func (r *UserRepository) CreateSectionItem(ctx context.Context, sectionitem models.SectionItem, user string, sectiontype string) (models.SectionItem, error) {
+	exists, err := r.UserExists(ctx, user)
+	if !exists {
+		print("User does not exist.")
+		return models.SectionItem{}, err
```

Comment:
returning one of these might be a lil more helpful here
`return errs.NotFound("user", "id", user)
`